porcupine/plugins/urls.py

```diff
@@ -29,8 +29,8 @@ def find_urls(text: tkinter.Text, start: str, end: str) -> Iterable[Tuple[str, s
         url = url.split("'")[0]
         url = url.split('"')[0]
 
-        open2close = {"(": ")", "{": "}", "<": ">"}
-        close2open = {")": "(", "}": "{", ">": "<"}
+        open2close = {"(": ")", "{": "}", "<": ">", "`": "`"}
+        close2open = {")": "(", "}": "{", ">": "<", "`": "`"}
```

Comment:
It might be better to add another `.split()`, so that it's handled just like any other quotes. I'm not sure if it makes a practical difference, but I'd like to avoid weird gotchas.

Reply:
I looked over several times to see which was better and thought I had tried every situation, but now I realized that the `.split` solution is still better, because with the open-close method you can get something like this:

![image](https://user-images.githubusercontent.com/77941087/124668855-c0ca6780-deb1-11eb-93d2-70c6271b1854.png)